dali/test/python/test_operator_coord_transform.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import nvidia.dali as dali
+import nvidia.dali.fn as fn
+from test_utils import check_batch, dali_type
+
+def make_param(kind, shape):
+    if kind == "input":
+        return fn.uniform(range=(0, 1), shape=shape)
+    elif kind == "scalar input":
+        return fn.reshape(fn.uniform(range=(0, 1)), shape=[])
+    elif kind == "vector":
+        return np.random.rand(*shape).astype(np.float32)
+    elif kind == "scalar":
+        return np.random.rand()
+    else:
+        return None
+
+def clip(value, type = None):
+    try:
+        info = np.iinfo(type)
+        return np.clip(value, info.min, info.max)
+    except:
+        return value
+
+def make_data_batch(batch_size, in_dim, type):
+    np.random.seed(1234)
+    batch = []
+    lo = 0
+    hi = 1
+    if np.issubdtype(type, np.integer):
+        info = np.iinfo(type)
+        # clip range to +/- 1000000 to prevent excessively large epsilons
+        lo = max(info.min / 2, -1000000)
+        hi = min(info.max / 2, 1000000)
+
+    for i in range(batch_size):
+        batch.append((np.random.rand(np.random.randint(1, 10000), in_dim)*(hi-lo) + lo).astype(type))
+    return batch
+
+def get_data_source(batch_size, in_dim, type):
+    return lambda: make_data_batch(batch_size, in_dim, type)
+
+def _run_test(device, batch_size, out_dim, in_dim, in_dtype, out_dtype, M_kind, T_kind):
```

Comment:
```suggestion
def _run_test(device, batch_size, out_dim, in_dim, in_dtype, out_dtype, m_kind, t_kind):
```

Reply:
I prefer to keep this aligned with the argument names.